omod/src/main/java/org/openmrs/module/fhir2/web/SmartConformance.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.web;
+
+import lombok.Data;
+
+@Data
+public class SmartConformance {
+	
+	private String authorization_endpoint;
```

Comment:
This doesn't have to be dealt with straight-away, but these should be camelCase, i.e., `authorizationEndpoint` rather than `authorization_endpoint`. We can then translate them to the appropriate name using an annotation, i.e.,

```java
@JsonProperty("authorization_endpoint")
private String authorizationEndpoint;
```